desktop/core/base_requirements.txt

```diff
@@ -1,4 +1,4 @@
-setuptools==70.0.0
+setuptools==78.1.1
```

Comment:
Instead of upgrading it here, I think it makes sense to upgrade it in `generate_requirements.py`?
Also did it work for you when you upgraded here? How did you test it out E2E?

@agl29 - Your views?

Reply:
Yes, you are right! I updated the generate_requirements.py. I can see setuptools using 78.1.1 version from pip list. Basic hue functionalities (querying, file uploads, importer) works fine.